packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -93,12 +93,14 @@ export class Dropdown extends BaseComponent<IDropdownProps, IDropdownState> {
           aria-expanded={ isOpen ? 'true' : 'false' }
           role='combobox'
           aria-label={ label }
+          aria-describedby={ id + '-option' }
           aria-activedescendant={ selectedIndex >= 0 ? (this._id + '-list' + selectedIndex) : (this._id + '-list') }
           >
           <span
+            id={ id + '-option' }
             className='ms-Dropdown-title'
             key={ selectedIndex }
-            aria-live={ disabled || isOpen ? 'off' : 'polite' }
+            aria-live={ disabled || isOpen ? 'off' : 'assertive' }
```

Comment:
1. screen reader: JAWS Version 17.0.2619
    browser: Internet Explorer 11
2. I'll move aira-live to div to see if it works for JAWS

Thank you !